tests/spread/general/extension-flask/task.yaml

```diff
@@ -25,6 +25,11 @@ execute: |
   
   # test the part merging
   docker run --rm --entrypoint /bin/python3 flask-extension -c "import pathlib;assert not pathlib.Path('/srv/flask/app/README').exists()"
+  
+  # test the default flask service
+  docker run --rm --name flask-extension-container -d -p 8137:8000 flask-extension
```

Comment:
we should remove this container in the `restore` step
`docker rm -f flask-extension-container`

Reply:
Added, thanks!